config/stack/manifests/icon.svg

```diff
@@ -1,29 +1,30 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 23.0.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 83.6 83.6" style="enable-background:new 0 0 83.6 83.6;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:#2AC6EA;}
-	.st1{fill:#FFFFFF;}
-</style>
-<rect x="0.2" y="0.2" class="st0" width="83.3" height="83.3"/>
-<g>
-	<g>
-		<g>
-			<path class="st1" d="M57.3,74.4H26.7c-8,0-14.5-6.5-14.5-14.5V36c0-1.1,0.9-2.1,2.1-2.1h55.4c1.1,0,2.1,0.9,2.1,2.1v23.9
-				C71.8,67.9,65.3,74.4,57.3,74.4z M16.4,38v21.9c0,5.7,4.6,10.3,10.3,10.3h30.6c5.7,0,10.3-4.6,10.3-10.3V38H16.4z"/>
-		</g>
-		<g>
-			<path class="st1" d="M49.2,63.5H34.8c-1.1,0-2.1-0.9-2.1-2.1v-7.2c0-5.1,4.2-9.3,9.3-9.3s9.3,4.2,9.3,9.3v7.2
-				C51.3,62.5,50.4,63.5,49.2,63.5z M36.8,59.3h10.3v-5.2c0-2.9-2.3-5.2-5.2-5.2s-5.2,2.3-5.2,5.2v5.2H36.8z"/>
-		</g>
-		<g>
-			<path class="st1" d="M69.7,30L69.7,30H58.6c-1.1,0-2.1-0.9-2.1-2.1c0-1.1,0.9-2.1,2.1-2.1l0,0h9.1V13.5h-6.9V17
-				c0,0.5-0.2,1.1-0.6,1.5c-0.4,0.4-0.9,0.6-1.5,0.6l0,0H47.6c-1.1,0-2.1-0.9-2.1-2.1v-3.5h-6.9V17c0,1.1-0.9,2.1-2.1,2.1H25.4
-				c-0.5,0-1.1-0.2-1.5-0.6s-0.6-0.9-0.6-1.5v-3.5h-6.9v12.4h9c1.1,0,2.1,0.9,2.1,2.1s-0.9,2.1-2.1,2.1h-11c-1.1,0-2.1-0.9-2.1-2.1
-				V11.4c0-1.1,0.9-2.1,2.1-2.1h11.1c1.1,0,2.1,0.9,2.1,2.1v3.5h7v-3.5c0-1.1,0.9-2.1,2.1-2.1h11.1c1.1,0,2.1,0.9,2.1,2.1v3.5h6.9
-				v-3.5c0-1.1,0.9-2.1,2.1-2.1H70c0.8,0.1,1.8,1,1.8,2.1v16.5c0,0.5-0.2,1.1-0.6,1.5S70.2,30,69.7,30z"/>
-		</g>
-	</g>
-</g>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="65px" height="65px" viewBox="0 0 65 65" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
is this a fixed icon size because of these fields? i.e. will this same icon not be able to scale to bigger sizes?

Reply:
these fields essentially set the preferential size but since it is an svg it can still be scaled infinitely